From 265d4dc3d1dc8592ec1122aa04bfb7d431a43d5d Mon Sep 17 00:00:00 2001
From: Thomas Baumgart <thb@net-bembel.de>
Date: Sun, 3 Oct 2021 09:41:17 +0200
Subject: [PATCH] Fix build with newer Gwenhywfar versions

Use our own DEPRECATED definition

(cherry picked from commit 4f8a333c2273c2564197dc622c77ad675ac73adc)
---
 kmymoney/plugins/kbanking/kbanking.h | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/kmymoney/plugins/kbanking/kbanking.h b/kmymoney/plugins/kbanking/kbanking.h
index 17ca59b16..b3e2c549b 100644
--- a/kmymoney/plugins/kbanking/kbanking.h
+++ b/kmymoney/plugins/kbanking/kbanking.h
@@ -30,16 +30,15 @@ class KBAccountSettings;
 // ----------------------------------------------------------------------------
 // Project Includes
 
+#include "banking.hpp"
 #include "kmymoneyplugin.h"
-#include "onlinepluginextended.h"
+#include "kmymoneyutils.h"
+#include "mymoney/onlinejobtyped.h"
 #include "mymoneyaccount.h"
 #include "mymoneykeyvaluecontainer.h"
-
-#include "mymoney/onlinejobtyped.h"
+#include "onlinepluginextended.h"
 #include "onlinetasks/sepa/sepaonlinetransfer.h"
 
-#include "banking.hpp"
-
 /**
   * This class represents the KBanking plugin towards KMymoney.
   * All GUI related issues are handled in this object.
@@ -158,7 +157,7 @@ private:
       *
       * @deprecated
       */
-    bool updateAccount(const MyMoneyAccount& acc) DEPRECATED;
+    bool updateAccount(const MyMoneyAccount& acc) KMYMONEY_DEPRECATED;
 
     /**
       * Trigger the password cache timer
-- 
2.33.0

